examples/notebooks/introduction_enlyze_python_sdk.ipynb

```diff
@@ -0,0 +1,2981 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "80593ce2-f90c-407f-92cc-12d04d8f75f3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def models_to_dataframe(models: list):\n",
+    "    \"\"\"Convert a list of enlyze models to a dataframe\"\"\"\n",
+    "    return pd.DataFrame([m.__dict__ for m in models])\n",
+    "    "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "id": "001cd893-d5a1-4c1a-a24a-b8c3468e03f8",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<div>\n",
+       "<style scoped>\n",
+       "    .dataframe tbody tr th:only-of-type {\n",
+       "        vertical-align: middle;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe tbody tr th {\n",
+       "        vertical-align: top;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>_id</th>\n",
+       "      <th>display_name</th>\n",
+       "      <th>address</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>49</td>\n",
+       "      <td>Platform</td>\n",
+       "      <td>Maybachstrasse 156</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1</th>\n",
+       "      <td>32</td>\n",
+       "      <td>Köln</td>\n",
+       "      <td>Heliosstrasse 6a</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2</th>\n",
+       "      <td>35</td>\n",
+       "      <td>Stuttgart</td>\n",
+       "      <td>Mahlestrasse 67</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "   _id display_name             address\n",
+       "0   49     Platform  Maybachstrasse 156\n",
+       "1   32         Köln    Heliosstrasse 6a\n",
+       "2   35    Stuttgart     Mahlestrasse 67"
+      ]
+     },
+     "execution_count": 8,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "models_to_dataframe(sites)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "39500780-f10d-40b8-b1b8-4fa931c42171",
+   "metadata": {},
+   "source": [
+    "We will pick a site by selecting an element from the list. Ideally, we would use the `_id` to filter for the desired site, since the order of the list could change."
```

Comment:
No, please don't use `_id`. It's a private attribute for a reason, so it is [not documented](https://enlyze-python.readthedocs.io/en/latest/models.html#enlyze.models.Site) on purpose. That being said, we might want to implement our own `__repr__()` on it, because later code of yours illustrates that when printing `Site` objects with pandas, the `_id` attribute is prominently shown as the first argument.

Is there an alternative narrative than can go without using `_id`? If not, what kind of attribute is missing for you, the user, to properly work with `Site` objects?

Reply:
b6ad651